posts/2023-02-14-liberty-tools-intellij-early-release.adoc

```diff
@@ -0,0 +1,113 @@
+// image::img/blog/FILE_NAME[IMAGE CAPTION ,width=70%,align="center"]
+// -------------------------
+// "[.img_border_light]" = This adds a faint grey border around the image to make its edges sharper. Use it around screenshots but not           
+// around diagrams. Then double check how it looks.
+// There is also a "[.img_border_dark]" class which tends to work best with screenshots that are taken on dark backgrounds.
+// Change "FILE_NAME" to the name of the image file. Also make sure to put the image into the right folder which is: img/blog
+// change the "IMAGE CAPTION" to a couple words of what the image is
+// // // // // // // //
+
+With the latest early release of Liberty Tools for IntelliJ IDEA - iterate fast with Liberty dev mode, easily edit Liberty configuration files and code with assistance for MicroProfile and Jakarta EE APIs, all from within the editor!
+
+Try it out today with IntelliJ IDEA 2022.2+, installable via the link:https://plugins.jetbrains.com/plugin/14856-liberty-tools/[Jetbrains marketplace].
+
+Check out the video below to see all the Liberty Tools for IntelliJ IDEA features:
+
+video::2T-ALsTGRY0[youtube]
+
+== What's New
+
+=== Type-ahead completion for Jakarta EE, MicroProfile and Liberty configuration files
+
+In this early release, we have added all new type-ahead completion assistance for your Liberty configuration files and Jakarta EE and MicroProfile APIs.
+
+Liberty configuration assistance provides code completion in Liberty `server.xml`, `server.env`, and `bootstrap.properties` files. For example, the completion shown below offers possible Liberty features you can configure in your `server.xml` file's `featureManager` block.
+[.img_border_light]
+image::img/blog/LT-intellij-0.0.8-LCLS-server-xml-completion.png[Liberty configuration assistance provides code completion in Liberty `server.xml`, `server.env`, and `bootstrap.properties` files.,width=70%,align="center"]
+
+Editing assistance provides code completion in configuration and application files for Jakarta EE and MicroProfile APIs. For example, the `rest_get` code completion shown below inserts a code snippet for a RESTful Web Services GET method.
+[.img_border_light]
+image::img/blog/LT-intellij-0.0.8-LSP4Jakarta-rest-completion.png[Editing assistance provides code completion in configuration and application files for Jakarta EE and MicroProfile APIs.,width=70%,align="center"]
+
+=== Navigate using the *Go to Action* menu
+
+If you prefer to work entirely through keyboard shortcuts, all Liberty actions can now be run through the IntelliJ *Go to Action* menu. You can access the menu either by pressing `Shift` twice, then selecting the Actions tab, or by pressing `Ctrl` + `Shift` + `A` / `Cmd` + `Shift` + `A`.
+
+[.img_border_light]
+image::img/blog/LT-intellij-0.0.8-liberty-go-to-action-view.png[Run Liberty actions using the keyboard with the IntelliJ Go To Action menu.,width=70%,align="center"]
+
+=== Run/Debug configuration integration
```

Comment:
```suggestion
=== Attach a debugger at startup
```

or similar, focus the heading on what the user can do w/ this info. Use imperative voice to be parallel with the other headings